pages/common/cabal.md

```diff
@@ -0,0 +1,28 @@
+# cabal
+
+> Command line interface to the Haskell Cabal infrastructure.
```

Comment:
how about "the Haskell Cabal infrastructure." --> "the Cabal Haskell package infrastructure", or something to that effect? I feel it would be clearer.